src/py-opentimelineio/opentimelineio/opentime.py

```diff
@@ -29,12 +48,17 @@ def to_timecode(rt, rate=None, drop_frame=None):
 
 
 def to_frames(rt, rate=None):
+    """Turn a :class:`~RationalTime` into a frame number."""
     return rt.to_frames() if rate is None else rt.to_frames(rate)
 
 
 def to_seconds(rt):
+    """Convert a :class:`~RationalTime` into float seconds"""
     return rt.to_seconds()
 
 
 def to_time_string(rt):
+    """Convert this timecode to time with microsecond, as formated in
+    `FFMPEG <https://trac.ffmpeg.org/wiki/Seeking#Timeunitsyntax>`_
```

Comment:
```suggestion
    Please note that the documentation here does not correspond to the implementation of ffmpeg: `FFMPEG <https://trac.ffmpeg.org/wiki/Seeking#Timeunitsyntax>`_
```

Reply:
I just removed the reference to this doc.